integration-tests/src/test/java/oracle/kubernetes/operator/BaseTest.java

```diff
@@ -237,6 +283,11 @@ public void testClusterScaling(Operator operator, Domain domain) throws Exceptio
         "Scale domain " + domain.getDomainUid() + " Up to " + replicas + " managed servers");
     operator.scale(domainUid, domainMap.get("clusterName").toString(), replicas);
 
+    if (domainUid.equalsIgnoreCase("domain1")) {
+      logger.info("Adding sleep ");
```

Comment:
Please add a code comment explaining why the sleep is needed when it wasn't before.  If you don't have an explaination, please have teh comment reference a Bug that tracks the issue...

Reply:
this is the open issue we are still trying to track down. I don't think the sleep is needed when the actual issue is tracked.

Reply:
OK - can you add a comment that discusses the problem, and references a tracking bug for the issue?

Reply:
Ok